src/hive/types.js

```diff
@@ -120,6 +125,24 @@ class BeehiveDirective extends SchemaDirectiveVisitor {
 
 }
 
+class BeehiveDeleteDirective extends SchemaDirectiveVisitor{
+
+  visitFieldDefinition(field, details) {
+    const target_type_name = this.args.target_type_name
+    const schema = this.schema
+    const table_config = this.schema._beehive.tables[target_type_name]
```

Comment:
table_config actually won't be available when this runs, you can only fetch it in resolve